packages/hooks/src/useSystemTheme.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react'
+
+export enum THEME {
+  light = 'light',
+  dark = 'dark',
+}
+
+export const useSystemTheme = (): THEME => {
+  const [systemTheme, setSystemTheme] = useState(THEME.light)
+
+  useEffect(() => {
+    try {
+      const mql = window.matchMedia('(prefers-color-scheme: dark)')
+
+      const setThemeFromQuery = ({ matches }: { matches: boolean }) => {
+        setSystemTheme(matches ? THEME.dark : THEME.light)
+      }
+
+      mql.addEventListener('change', setThemeFromQuery)
+      setThemeFromQuery(mql)
+
+      return () => {
+        mql.removeEventListener('change', setThemeFromQuery)
+      }
+    } catch (error) {
+      return
```

Comment:
What do you think is it necessary to leave catch block empty with no error logging?

Reply:
you can:
try {
  ...
} catch {
  /* noop */
}

Reply:
> you can: try { ... } catch { /* noop */ }

What does this achieve?